docs/API.md

```diff
@@ -18,12 +18,16 @@ There are a number of methods that a DB storage backend should implement:
     * .createSessionToken(tokenId, sessionToken)
     * .updateSessionToken(tokenId, sessionToken)
     * .sessionToken(id)
+    * .sessionTokenVerified(tokenId)
```

Comment:
IIUC, the only difference between `sessionToken` and `sessionTokenVerified` is the `Verified` variant returns the `tokenVerificationId`. Why require a separate endpoint for the extra bit of data?

The reason I'm asking is because /recovery_email/resend_code uses the `sessionToken` strategy to fetch the sessionToken. That endpoint needs to know `sessionToken.tokenVerificationId` to send the correct code in the email, if the `tokenVerificationId` is different to the `sessionToken.emailCode`.